liberty-maven-webapp-parent/pom.xml

```diff
@@ -50,14 +50,12 @@
                             <goals>
                                 <goal>stop-server</goal>
                             </goals>
-                            <configuration>
-                                <failOnError>false</failOnError>
-                            </configuration>
                         </execution>
                         <execution>
                             <id>create-server</id>
                             <phase>prepare-package</phase>
                             <goals>
+                                <goal>install-server</goal>
```

Comment:
For command line (mvn install), we still need to call create-server goal to create the server.  But the question I have to myself is whether I need to generate the xml file if create-server goal is called without install-server is being called first.  Right now, I am generating the xml file in both goals.

The goal will be automatically called during import or maven project update by the m2e plugin if it is in both the pom and the m2e lifecycle mapping